docs/reference/objectmodel/preview-requirement-set/office.md

```diff
@@ -29,7 +29,7 @@ The Office namespace provides shared interfaces that are used by add-ins in all
 
 [context](office.context.md): Provides shared interfaces from the Office Add-ins API's context namespace for use in the Outlook add-in API.
 
-[MailboxEnums](/javascript/api/outlook/office.mailboxenums.attachmenttype): Includes the ItemType, EntityType, AttachmentType, RecipientType, ResponseType, and ItemNotificationMessageType enumerations.
+[MailboxEnums](/javascript/api/outlook/office.mailboxenums.AttachmentContentFormat): Includes a number of enumerations, for example, `ItemType`, `EntityType`, `AttachmentType`, `RecipientType`, `ResponseType`, and `ItemNotificationMessageType`.
```

Comment:
Links should be lowercase.

Reply:
Is that because we'll get a 404 or for consistency?

Reply:
Consistency, but I some sites are case-sensitive.